frontend/src/stores/OptionsStore.js

```diff
@@ -41,6 +41,10 @@ class OptionsStore {
     @computed get getDatasetType() {
         return this.rootStore.mainStore.dataset_type;
     }
+
+    @computed get canAddNewOption() {
+        return this.optionsList.length > 2;
```

Comment:
the variable name doesnt match the logic here. you can add a new option if the length is <=3 right?

Reply:
switched the logic to  length<3. if length <= 3 you can still add one more until the flag becomes false.